### ISSUE TITLE
JupyterLab settings not respected

### ISSUE THREAD

**Issue by psychemedia** [ploomber/jupysql#1010]
### What happens?

Initialising a new jupyterlab environment with default settings to disable the display of the `Share notebook` and `Format SQL` toolbar buttons, the settings are not respected although the settings editor describes them correctly.

<img width="423" alt="image" src="https://github.com/ploomber/jupysql/assets/82988/903ee080-5f1c-46f9-aa48-7d1ef158dc08">

Enabling then disabling the setting from the settings panel does then correctly hide the buttons.

### To Reproduce

I used the following settings:

In `.jupyter/lab/user-settings/ploomber-extension/settings.jupyterlab-settings`:

```json
{
    "showShareNotebook": false
}
```

In `.jupyter/lab/user-settings/jupysql-plugin/settings.jupyterlab-settings`:

```json
{
    "showFormatSQL": false
}
```

Start the jupyter server, open a notebook, and the buttons are displayed.

### OS:

Linux Ubuntu

### JupySQL Version:

0.10.10

### Full Name:

Tony Hirst

### Affiliation:

The Open University

**Comment by edublancas** [comment]
thanks, we'll look at this

in the meantime, you can try uninstalling jupysql-plugin after installing jupysql, that should remove both buttons

**Comment by edublancas** [comment]
Note: we fixed the "share notebook" button. we're working on a fix for the "format SQL" (they're in different repos)

**Comment by edublancas** [comment]
the "format sql" issue has been fixed in jupysql-plugin 0.4.4
